apps/api/app/helpers/message_helpers.py

```
@@ -97,6 +100,28 @@ async def _get_gaia_knowledge_section(query: str) -> str:
 
 
+async def _get_tracked_todos_section(user_id: str) -> str:
+    """Fetch active tracked-todo summary with 60s Redis cache."""
+    cache_key = f"tracked_todos:summary:{user_id}"
+
+    try:
+        cached = await get_cache(cache_key)
+        if cached:
+            return cached if isinstance(cached, str) else str(cached)
+    except Exception:
+        pass  # Cache miss is fine
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)